app/directives/ui/datatables/containers-datatable/containersDatatable.html

```diff
@@ -202,7 +202,7 @@
                 </div>
               </td>
               <td>{{ item.StackName ? item.StackName : '-' }}</td>
-              <td><a ui-sref="image({ id: item.Image })">{{ item.Image | hideshasum }}</a></td>
+              <td><a ui-sref="image({ id: item.Image })">{{ item.Image | trimshasum | hideshasum }}</a></td>
```

Comment:
I think that the hideshasum filter is not required here, it's only useful for services.

Reply:
Yes, remove it anyway as it should be part of the filter that trim the prefix and then truncate the image name (equivalent of the Docker CLI).

Reply:
Sorry I completely misread your first comment.